docs/basics.adoc

```diff
@@ -1476,6 +1492,26 @@ By default, JanusGraph uses the Astyanax library to connect to Cassandra cluster
 
 When numerous clients are connecting to Elasticsearch, it is likely that an `OutOfMemoryException` occurs. This is not due to a memory issue, but to the OS not allowing more threads to be spawned by the user (the user running Elasticsearch). To circumvent this issue, increase the number of allowed processes to the user running Elasticsearch. For example, increase the `ulimit -u` from the default 1024 to 10024.
 
+=== Dropping a Database
+
+To drop a database using the Gremlin Console you can call `JanusGraphFactory.drop(graph)`. The graph you want to drop needs to be defined prior to running the drop method.
+
+With ConfiguredGraphFactory
+[source, gremlin]
+----
+graph = ConfiguredGraphFactory.open("example");
+JanusGraphFactory.drop(graph);
+----
+
+With JanusGraphFactory
+[source, gremlin]
+----
+graph = JanusGraphFactory.open('path/to/configuration.properties')
+JanusGraphFactory.drop(graph);
+----
+
+Note that if multiple clients are connecting to the graph that has been dropped it will be necessary to run `graph.close()`, and then reopen the graph, on all nodes to clear the cached(stale) versions.
```

Comment:
This statement isn't exactly correct. The graph needs to be removed from the cache from every node in the JG cluster, and I believe that is fulfilled by executing `JGF/CGF.close()` on each node in the cluster.

Reply:
The goal for me with this PR was trying to help improve the User on-boarding experience, by covering gaps for basic operations. Since this is just covering basics.adoc would it make sense to open an issue to add the clustering operations(like how do you cluster) to the advanced topics.

My reasoning being from what I've read of the current JanusGraph documentation there really isn't any reference to clustering other than on the backend. The closest I know of is the instructions for running [Separate servers for WebSockets and HTTP](http://docs.janusgraph.org/latest/server.html#_websocket_versus_http). By nodes the JG cluster to you mean all Gremlin server pointed at the same backend?

I'm more than happy to be wrong about the lack of a clustering concept in the docs and will gladly read through and link to any existing docs I can be pointed at.

Reply:
I updated the note to reference using multiple Gremlin Server instances and in such a way that it won't need to be patched in 0.3.0.

Reply:
`graph.close()` does not remove the graph from the cache